### PR TITLE
first steps

### PR DESCRIPTION
**Pull request by moshekruger**

### CODE REVIEW

File: docs/learn-about-neptune/ui.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      :width: 250
+
+- **Custom views**: You can create a custom view of the dashboard, for example, by filtering selected rows, columns and columns order, sorting and width. Then you can save the view and quickly return to it by selecting the desired view from the list of views.
+
+   .. image:: /_static/images/core-concepts/view_list.png
+      :target: /_static/images/core-concepts/view_list.png
+      :alt: View list
+      :width: 400
+
+- **Choosing columns**: Columns in the dashboard are configurable. All types of data logged to Neptune can be seen as columns in the dashboard, for example parameters, metrics or tags.
```
Comment (pitercl) [inline docs/learn-about-neptune/ui.rst]: `All types of data logged to Neptune can be seen` - most data cannot be seen here. Only metrics, text logs, properties and parameters.

File: docs/learn-about-neptune/ui.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+- **Grouping**: You can group experiments by one or more column(s). The dashboard displays the selected columns, allowing you to make in-group and across-groups analysis of the experiments.
```
Comment (pitercl) [inline docs/learn-about-neptune/ui.rst]: Maybe we can write here that each group is represented by its first experiment (according to sorting order)?
And also, that each group (after opening it) shows at most 10 experiments and all experiments can be viewed by clicking `Show all`.

File: docs/learn-about-neptune/ui.rst [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+
+   .. image:: /_static/images/core-concepts/compare_experiments_select.png
+      :target: /_static/images/core-concepts/compare_experiments_select.png
+      :alt: Compare experiments table
+
+
+- **Overlaid charts**: In the comparison view, all metrics with the same name are placed on a single chart with one curve per experiment. The rich and customizable legend lets you display additional information.
+
+   .. image:: /_static/images/core-concepts/charts_legend.png
```
Comment (pitercl) [inline docs/learn-about-neptune/ui.rst]: This image doesn't show the actual legend, only a menu for choosing its elements. I'd much prefer showing the actual legend.

File: docs/learn-about-neptune/ui.rst [hunk 4]
```diff
@@ -0,0 +1,137 @@
+Working with the Neptune UI
+===========================
+
+The Neptune client is a browser-enabled app that lets you visualize and browse experiments.
```
[the remark this answers is not in the document]
Comment (jakubczakon) [reply docs/learn-about-neptune/ui.rst]: Yeah, it should simply be named Neptune.
Neptune client feels like the lib I am using in Python to send things to the Neptune app.

File: docs/learn-about-neptune/ui.rst [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+There are several ways to organize your experiments:
+
+- **Tags**: You can create tag(s), which you assign to experiments. Later, you can quickly filter by these tags.
+
+   .. image:: /_static/images/core-concepts/tag_chooser.png
+      :target: /_static/images/core-concepts/tag_chooser.png
+      :alt: Tag chooser
+      :width: 250
+
+- **Custom views**: You can create a custom view of the dashboard, for example, by filtering selected rows, columns and columns order, sorting and width. Then you can save the view and quickly return to it by selecting the desired view from the list of views.
```
[the remark this answers is not in the document]
Comment (jakubczakon) [reply docs/learn-about-neptune/ui.rst]: I think going with a simple:

"you can filters rows by parameter or metric values and select a subset of useful columns" 

would do, right?

File: docs/learn-about-neptune/ui.rst (hunk 1 above)
Comment (jakubczakon) [reply docs/learn-about-neptune/ui.rst]: That is correct **some** data logged to Neptune can be used as columns. All data can be seen in the single experiment view.

File: docs/learn-about-neptune/ui.rst (hunk 3 above)
Comment (jakubczakon) [reply docs/learn-about-neptune/ui.rst]: Charts legend as in what can be seen on hover.oCharst
You need to first select a couple of metrics and/or parameters and then those will be seen on hover.

File: docs/learn-about-neptune/ui.rst [hunk 6]
```diff
@@ -83,21 +90,22 @@ Simply select experiments and click **Compare**:
       :alt: Compare experiments table
 
 
-- **Overlaid charts**: In the comparison view, all metrics with the same name are placed on a single chart with one curve per experiment. The rich and customizable legend lets you display additional information.
+- **Overlaid charts**: In the comparison view, all metrics with the same name are placed on a single chart with one curve per experiment. The customizable legend lets you select additional metrics and/or parameters to display. When hovering withe the mouse over a particular area, the values for the selected metrics are displayed below:
```
Comment (pitercl) [inline docs/learn-about-neptune/ui.rst]: Typo in `withe`.

File: docs/learn-about-neptune/team-management.rst [hunk 7]
```diff
@@ -1,14 +1,138 @@
...
+If you have :ref:`created an organization <how-to_team-management_create-organization>`,
+you can invite people to it. Two roles are available: Admin and Member.
+
+The following table summarizes the permissions available for the organization roles.
+
+.. csv-table::
+   :header: "Permission","Admin","Member"
+   :widths: 20, 20, 20
+
+    Access level in subscription management,edit,none
```
Comment (pitercl) [inline docs/learn-about-neptune/team-management.rst]: Those `Access levels` are unreadable, to me.
Wouldn't it be more straightforward to just say

```
Managing subscription (including invoice data),yes,no
Managing organization members,yes,no
Creating and deleting projects,yes,no
```

File: docs/learn-about-neptune/team-management.rst [hunk 8]
```diff
@@ -1,14 +1,138 @@
...
+You can access organization settings by clicking **Settings**.
+
+.. figure:: ../_static/images/core-concepts/org-settings.png
+   :target: ../_static/images/core-concepts/org-settings.png
+   :alt: organization settings button
+
+
+.. note::
+
+    - An Admin of an organization is also the Owner of all projects, by default.
```
Comment (pitercl) [inline docs/learn-about-neptune/team-management.rst]: Please remove ", by default" - it's actually non-negotiable. You cannot change the role of an organization Admin in a project (or remove him/her from it).

File: docs/learn-about-neptune/team-management.rst [hunk 9]
```diff
@@ -1,14 +1,138 @@
...
+    - Project Owners can grant project ownership to anyone in the organization.
```
Comment (pitercl) [inline docs/learn-about-neptune/team-management.rst]: Can we add a similar statement regarding organization Admins?

```
    - Organization Admins can grant the Admin role to anyone in the organization.
```

File: docs/learn-about-neptune/team-management.rst [hunk 10]
```diff
@@ -1,14 +1,138 @@
...
+.. csv-table::
+   :header: "Permission","Owner","Contributor","Viewer"
+   :widths: 20, 20, 20, 20
+
+    Access level in project settings,edit,none,none
+    Access level to all experiments Notebooks and Wiki,edit,edit,view
+    Access level to people in project (under Settings tab),edit,view,view
+    Can run experiments,yes,yes,no
+    Can add notebooks and make checkpoints,yes,yes,no
+    Can remove project,yes,no,no
```
Comment (pitercl) [inline docs/learn-about-neptune/team-management.rst]: Project Owners cannot remove projects. Only organization Admins can do that. I suggest removing this row from the table.

File: docs/learn-about-neptune/team-management.rst [hunk 11]
```diff
@@ -1,14 +1,138 @@
...
+
+Private projects
+^^^^^^^^^^^^^^^^
+Only people added to the project can see it. The project :ref:`Owner <core-concepts_user-roles_project-owner>` can manage who has access to the project in the Settings view.
+
+Here is an example view, where the project Owner can manage project members:
+
+.. image:: ../_static/images/core-concepts/invite-to-project.png
+   :target: ../_static/images/core-concepts/invite-to-project.png
+   :alt: Invite user to the project
```
Comment (pitercl) [inline docs/learn-about-neptune/team-management.rst]: This screenshot shows "Invites left", which is a concept long gone from Neptune.

File: docs/learn-about-neptune/team-management.rst [hunk 12]
```diff
@@ -1,14 +1,138 @@
...
+
+Public projects
+^^^^^^^^^^^^^^^
+Public projects are freely available to view by everyone who has access to the Internet.
+
+Also, in case of individual organizations, you can invite collaborators to public projects only.
```
Comment (pitercl) [inline docs/learn-about-neptune/team-management.rst]: Can we move this sentence after `There are private and public projects.`? I think it's a more appropriate place.

File: docs/learn-about-neptune/team-management.rst [hunk 13]
```diff
@@ -1,14 +1,138 @@
...
+.. _core-concepts_user-roles_project-owner:
+
+The following table summarizes the permissions available for the three project roles: Owner, Contributor and Viewer.
+
+.. csv-table::
+   :header: "Permission","Owner","Contributor","Viewer"
+   :widths: 20, 20, 20, 20
+
+    Access level in project settings,edit,none,none
+    Access level to all experiments Notebooks and Wiki,edit,edit,view
```
Comment (pitercl) [inline docs/learn-about-neptune/team-management.rst]: Missing comma after "experiments".

File: docs/learn-about-neptune/team-management.rst [hunk 14]
```diff
@@ -1,14 +1,138 @@
...
+    - Project Owners can grant project ownership to anyone in the organization.
+    - An organization must have at least one Admin, so the last remaining Admin cannot be removed from the organization.
+
+Roles in a project
+^^^^^^^^^^^^^^^^^^
+.. _core-concepts_user-roles_project-owner:
+
+The following table summarizes the permissions available for the three project roles: Owner, Contributor and Viewer.
+
+.. csv-table::
```
Comment (pitercl) [inline docs/learn-about-neptune/team-management.rst]: This table is confusing, in my opinion.

The concept is actually quite simple:

There are 3 groups of resources in a project:
1. Its content (experiments, notebooks, wiki)
2. Project's members
3. Project's properties

Ad 1 (content):
Viewer can view anything.
Contributor can edit anything (add, run, delete, modify, etc).
Owner is the same as Contributor with regard to content.

Ad 2 (members):
Viewer can view
Contributor can view
Owner can modify (invite, remove, change roles)

Ad3 (settings):
Viewer cannot access (view or edit)
Contributor cannot access (view or edit)
Owner can modify (change visibility, name)

**Pull request merged**